### PR TITLE
5 get api articles

### PR DESCRIPTION
**Pull request by WolfieKnee**
This PR completes ticket 5. 
Error handling for bad endpoint is already in place. 

Also addresses feedback from previous tickets, including:
- adding a test for consoles
- refactoring of error handling

### CODE REVIEW

**Comment by WolfieKnee** [comment]
After submitting this PR I found an error in app.js:
`13 } = require("./models/errors");`
should be: 
`13 } = require("./models/errors");`

I then got in a mess with merge conflicts trying to correct this but did so directly on the main branch.

File: __tests__/app.test.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ const seed = require("../db/seeds/seed");
 beforeEach(() => seed(data));
 afterAll(() => db.end());
```
Comment (nc-richard-neat) [inline __tests__/app.test.js:9]: Cool idea here to make sure we don't have any console.logs left in the code. We could use the beforeEach and afterAll we already have to keep this a bit cleaner and more DRY>

**Review by nc-richard-neat — COMMENTED** [review]
Really nice :)

File: __tests__/app.test.js (hunk 1 above)
Comment (WolfieKnee) [reply __tests__/app.test.js:9]: Good call - I got in a mess here for a while with the tests hanging, then realised that I needed to return the `seed()`

**Pull request merged**